common/src/main/java/com/wynntils/models/containers/type/SearchableContainerType.java

```diff
@@ -51,4 +69,11 @@ public static SearchableContainerType getContainerType(StyledText title) {
 
         return null;
     }
+
+    public Stream<ItemStack> getSearchableItems(AbstractContainerScreen<?> screen) {
+        return screen.getMenu().slots.stream()
+                // see ChestMenu
+                .filter(slot -> isSearchableSlot.test((slot.x - 8) / 18, (slot.y - 18) / 18))
```

Comment:
This looks a bit odd, but maybe the slot id was not a good way of determining if it was searchable or not?

Reply:
I honestly have no idea what the -8 does, as well as 18 (2*9, 2 rows)?

Reply:
The x and y components of Slot are in screen coordinates relative to the upper left corner of the container screen. You can see where these numbers come from in ChestMenu

Reply:
And why do we compare to rendered coordinates in logic that is not about rendering? Refactor this class to use `Slot.index`, otherwise it is really sketch.

Reply:
(Otherwise this will break really easy if a mod, or a MC update moves slots by 1 pixel..)

Reply:
I agree with bolyai that using screen coordinates is not the best way to do this. We really want to use the slot index id.

However, in my experience, it is not obvious how to express "3rd row, 2nd column" as a slot id, or contrary, what position in the container slot id 13 means.

But I think the solution to this is instead to create a function like `getSlotId(int row, int column)` in `InventoryUtils`, and use it here.